src/ansi.rs

```diff
@@ -445,7 +445,7 @@ impl Mode {
                 1049 => Mode::SwapScreenAndSetRestoreCursor,
                 2004 => Mode::BracketedPaste,
                 _ => {
-                    trace!("[unhandled] mode={:?}", num);
+                    trace!("from_primitive [unhandled] mode={:?}", num);
```

Comment:
To be in line with the other suggestion to keep trace/info message formats the same, I think this can be rewritten to be a little more human-readable.

One idea:
```
Unhandled primitive mode: {:?}
```

That conveys the same information, but it's a clear message that can be easily read. If the purpose is to search for the message in the Alacritty source code, this shouldn't make things more difficult either.